quarkchain/cluster/shard_state.py

```diff
@@ -106,8 +106,8 @@ def __get_current_tx(self):
 
                 # Perform x-shard from root chain coinbase
                 return CrossShardTransactionDeposit(
-                    tx_hash=self.rblock.header.get_hash(),
-                    from_address=self.rblock.header.coinbase_address,
+                    tx_hash=bytes(32),
```

Comment:
May be we should use sha3(rootblock.get_hash() + "coinbase")?  This will make sure each tx_hash is unique.

Reply:
any benefit for making it unique? because it will again break the indexer, which by default regards `tx_hash == bytes(32)` as root block coinbase reward

Reply:
the reason for my change is to find a way to differentiate between real xshard deposit and root block coinbase, because the indexing server should only index the former

Reply:
Yes, but in general, one txhash = one tx, while now we have one txhash = a list of txs.  For index server, do you think we can resort other way to detect (but probably still need to show the coinbase tx properly).